tests/wallet/cat_wallet/test_trades.py

```diff
@@ -1750,7 +1750,7 @@ async def test_trade_conflict(three_wallets_prefarm):
     await time_out_assert(15, get_trade_and_status, TradeStatus.PENDING_CONFIRM, trade_manager_taker, tr1)
     # pushing into mempool while already in it should fail
     tr2, txs2 = await trade_manager_trader.respond_to_offer(offer, peer, DEFAULT_TX_CONFIG, fee=uint64(10))
-    await trade_manager_taker.wallet_state_manager.add_pending_transactions(txs2)
+    await trade_manager_trader.wallet_state_manager.add_pending_transactions(txs2)
```

Comment:
typikcally the counterpart to "taker" is "maker". I imagine that's what "trader" refers to here. is that right?
It's not intuitive why this change makes a difference or why it makes sense. Presumably it's supposed to be the *taker* that submits the transaction

Reply:
oh, there are 3 wallets involved here, and there is both a taker and maker.

Reply:
Yeah poor naming forsure but the purpose of this change is so that the transactions being created are actually pushed to the correct wallet state manager.